crates/evm/execution-errors/src/lib.rs

```diff
@@ -212,3 +226,15 @@ impl InternalBlockExecutionError {
         Self::Other(msg.to_string().into())
     }
 }
+
+impl From<PruneSegmentError> for InternalBlockExecutionError {
+    fn from(error: PruneSegmentError) -> Self {
+        Self::Pruning(error)
+    }
+}
+
+impl From<ProviderError> for InternalBlockExecutionError {
+    fn from(error: ProviderError) -> Self {
+        Self::LatestBlock(error)
+    }
+}
```

Comment:
replace all the from impls with [`derive_more::From`](https://jeltef.github.io/derive_more/derive_more/from.html)

Reply:
done